File: SQLITE3Demo/SQLITE3Demo/ViewController.m
//
//  ViewController.m
//  SQLITE3Demo
//
//  Created by 刘康 on 16/2/26.
//  Copyright © 2016年 刘康. All rights reserved.
//

#import "ViewController.h"
#import "DataHelper.h"

@interface ViewController ()
@property (strong, nonatomic)DataHelper* dataHelper;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    self.dataHelper = [DataHelper new];
}

- (IBAction)tap:(UIButton *)sender {
    if (sender.tag == 0) {
        [self.dataHelper openDatabase];
    } else if (sender.tag == 1) {
        [self.dataHelper insertData];
    } else {
        [self.dataHelper readData];
    }
}


@end

File: AnimateTextView/AnimateTextView/ViewController.swift
//
//  ViewController.swift
//  AnimateTextView
//
//  Created by Origheart on 2017/5/4.
//  Copyright © 2017年 origheart. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var animateTextView: AnimateTextView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
    }
    
    @IBAction func animate(_ sender: UIButton) {
        animateTextView.textList = ["开始筛选案例", "英国 硕士", "北京大学", "电子信息工程专业", "筛选完成"]
        animateTextView.animate(duration: 5)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

